homeassistant/components/gdacs/sensor.py

```diff
@@ -107,6 +107,11 @@ def state(self):
         """Return the state of the sensor."""
         return self._total
 
+    @property
+    def unique_id(self) -> Optional[str]:
+        """Return a unique ID containing latitude/longitude."""
+        return self.name
```

Comment:
Please change here too, to use `config_entry.unique_id`.

Reply:
OK